endpointmanager/pkg/chplendpointquerier/abeosolutionsparser.go

```diff
@@ -0,0 +1,57 @@
+package chplendpointquerier
+
+import (
+	"io"
+	"strings"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/helpers"
+
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func AbeoSolutionsCSVParser(CHPLURL string, fileToWriteTo string) {
+	var lanternEntryList []LanternEntry
+	var endpointEntryList EndpointList
+
+	csvFilePath := "./FHIRServiceURLs.csv"
+
+	csvReader, file, err := helpers.QueryAndOpenCSV(CHPLURL, csvFilePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	for {
+		rec, err := csvReader.Read()
```

Comment:
I think for Abeo Solutions, looking at the CSV, we just want the first endpoint, since the rest are either test or OAUTH endpoints?